### PR TITLE
Update Tado device tracker

### PR DESCRIPTION
**Pull request by erwindouna**
## Proposed change

Update the paragraph that explains how the Tado presence detection works. This is now migrated in the new mechanism that will automatically add devices, if the geolocation setting is enabled in the Tado app.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102610
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by erwindouna** [comment]
@MartinHjelmare here's the new PR.

File: source/_integrations/tado.markdown [hunk 1]
```diff
@@ -48,18 +48,9 @@ It currently supports presenting the current temperature, the setting temperatur
...
-# Example configuration.yaml entry for Tado
-device_tracker:
-  - platform: tado
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-    home_id: YOUR_HOME_ID
-```
+By default the Tado device tracker will track all devices known to Tado associated with your home. The Tado app needs to have the `Geolocation` permission enabled for the device to be tracked.
 
 {% configuration %}
```
Comment (MartinHjelmare) [inline source/_integrations/tado.markdown]: We can remove the configuration section and how to get the home id.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — DISMISSED** [review]
Thanks!

File: source/_integrations/tado.markdown [hunk 2]
```diff
@@ -48,74 +48,15 @@ It currently supports presenting the current temperature, the setting temperatur
...
-  description: The id of your home of which you want to track devices. If provided, the Tado device tracker will track *all* devices known to Tado associated with this home. See below how to find it.
-  required: false
-  type: integer
-{% endconfiguration %}
-
-After configuration, your device has to be at home at least once before showing up as *home* or *away*.
+Your device has to be at home at least once before showing up as *home* or *away*.
 Polling Tado API for presence information will happen at most once every 30 seconds.
 
 See the [device tracker integration page](/integrations/device_tracker/) for instructions how to configure the people to be tracked. Beware that the Tado (v2) API does not provide GPS location of devices, only a bearing, therefore Home Assistant only uses `home`/`not-home` status.
```
Comment (MartinHjelmare) [inline source/_integrations/tado.markdown]:
```suggestion
Beware that the Tado (v2) API does not provide GPS location of devices, only a bearing, therefore Home Assistant only uses `home`/`not-home` status.
```

**Review by MartinHjelmare — COMMENTED** [review]
Remove stale reference to device tracker configuration.

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/tado.markdown [hunk 3]
```diff
@@ -48,73 +48,14 @@ It currently supports presenting the current temperature, the setting temperatur
 
 The `tado` device tracker is using the [Tado Smart Thermostat](https://www.tado.com/) and its support for person presence detection based on smartphone location by geofencing.
```
Comment (MartinHjelmare) [inline source/_integrations/tado.markdown]:
```suggestion
The Tado device tracker is using the [Tado Smart Thermostat](https://www.tado.com/) and its support for person presence detection based on smartphone location by geofencing.
```

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**